modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentFolderResourceTest.java

```diff
@@ -110,4 +137,36 @@ protected String[] getIgnoredEntityFieldNames() {
 		return testDepotEntry.getDepotEntryId();
 	}
 
+	@Override
+	protected StructuredContentFolder
+			testPutSiteStructuredContentFolderByExternalReferenceCode_createStructuredContentFolder()
+		throws Exception {
+
+		return _randomStructuredContentFolder();
```

Comment:
I don't think Brian will pass this because it doesn't start with `get` and `random` is not something we use normally. Just call it `getStructuredContentFolder`.

Reply:
@4lejandrito if you see in the last part of the method: `_createStructuredContentFolder` it only create a new random StructuredContentFolder object to use in the test case

Reply:
Ok I see there is a patter of `random*` methods in headless tests. Then it's fine.